portfolio/src/main/webapp/index.html

```diff
@@ -27,11 +27,17 @@ <h1>Heidi Johnson's Portfolio</h1>
         </div>
       </section>
       <section class="comments">   
-        <form id="addcomment" action="/data" method="POST">
-          <label for="comment">Add Comment:</label>
-          <input type="text" name="comment">
-          <input type="submit" />
-        </form>   
+        <div id="newcomment">
+          <p>Add Comment</p>
+          <form id="addcomment" action="/data" method="POST">
+            <label for="name">Name: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
```

Comment:
You might want to consider using padding on your label instead of adding lots of no break space characters